src/track/globaltrackcache.h

```diff
@@ -32,32 +32,53 @@ class /*interface*/ GlobalTrackCacheRelocator {
     virtual ~GlobalTrackCacheRelocator() {}
 };
 
+typedef void (*deleteTrackFn)(Track*);
+
 class GlobalTrackCacheEntry final {
     // We need to hold two shared pointers, the deletingPtr is
     // responsible for the lifetime of the Track object itselfe.
     // The second one counts the references outside Mixxx, if it
     // is not longer referenced, the track is saved and evicted
     // from the cache.
   public:
+    class TrackDeleter {
+    public:
+        explicit TrackDeleter(deleteTrackFn deleteTrack = nullptr)
+                : m_deleteTrack(deleteTrack) {
+        }
+
+        void operator()(Track* pTrack) const;
+
+    private:
+        deleteTrackFn m_deleteTrack;
```

Comment:
`m_typeName`? The name of the function is `deleteTrack`, the corresponding type is `deleteTrackFn`.

Reply:
I have stumbled over this because it can also be bool m_deleteTrack; 
The idea was to make it obvious from the variable name that this is a function pointer.